insights-client.spec

```diff
@@ -62,17 +63,71 @@ Resource Optimization service upon modifying ros_collect parameter to True.
 
 
 %build
-%{meson} \
-    -Dpython=%{__python3} \
-%if (0%{?rhel} && 0%{?rhel} < 10)
-    -Dredhat_access_insights=true \
-%endif
-    %{nil}
-%{meson_build}
+sed -e "s|@PYTHON@|%{__python3}|g" -e "s|@pythondir@|%{python3_sitelib}|g" \
+    src/insights-client.in > insights-client
 
+sed -e "s|@PACKAGE@|%{name}|g" \
+    -e "s|@PACKAGE_VERSION@|%{version}|g" \
+    -e "s|@CORE_SELINUX_POLICY@||g" \
+    src/insights_client/constants.py.in > src/insights_client/constants.py
 
 %install
-%{meson_install}
+# ./src/
+install -d -m 755 %{buildroot}%{_bindir}/
+install -m 755 insights-client %{buildroot}%{_bindir}/
+install -d -m 755 %{buildroot}%{python3_sitelib}/insights_client/
+cp -pr src/insights_client/* %{buildroot}%{python3_sitelib}/insights_client/
+rm -rf %{buildroot}%{python3_sitelib}/insights_client/tests/
+rm -f %{buildroot}%{python3_sitelib}/insights_client/constants.py.in
+
+# ./data/
+install -d -m 755 %{buildroot}%{_sysconfdir}/insights-client/
+install -m 644 data/cert-api.access.redhat.com.pem %{buildroot}%{_sysconfdir}/insights-client/cert-api.access.redhat.com.pem
+install -m 644 data/insights-client.conf %{buildroot}%{_sysconfdir}/insights-client/insights-client.conf
+install -m 644 data/insights-client.motd %{buildroot}%{_sysconfdir}/insights-client/insights-client.motd
+install -m 644 data/redhattools.pub.gpg %{buildroot}%{_sysconfdir}/insights-client/redhattools.pub.gpg
+
+# ./data/logrotate.d/
+install -d -m 755 %{buildroot}%{_sysconfdir}/logrotate.d/
+install -m 644 data/logrotate.d/insights-client %{buildroot}%{_sysconfdir}/logrotate.d/insights-client
+
+# ./data/systemd/
+install -d -m 755 %{buildroot}%{_unitdir}/
+install -d -m 755 %{buildroot}%{_presetdir}/
+install -m 644 data/systemd/insights-client-boot.service %{buildroot}%{_unitdir}/insights-client-boot.service
+install -m 644 data/systemd/insights-client.service %{buildroot}%{_unitdir}/insights-client.service
+install -m 644 data/systemd/insights-client.timer %{buildroot}%{_unitdir}/insights-client.timer
+install -m 644 data/systemd/80-insights.preset %{buildroot}%{_presetdir}/80-insights.preset
+install -m 644 data/systemd/insights-client-results.service %{buildroot}%{_unitdir}/insights-client-results.service
+install -m 644 data/systemd/insights-client-results.path %{buildroot}%{_unitdir}/insights-client-results.path
```

Comment:
Do we need to list the files here explicitly? Would `data/systemd/*` behave the same?